sdk/core/abort-controller/README.md

```diff
@@ -1,15 +1,16 @@
 # Azure Abort Controller library for JS
 
-The `@azure/abort-controller` package provides `AbortController` and `AbortSignal` classes that are compatible
-with the `AbortController` built into modern browsers and the `AbortSignal` used by `fetch`.
-These classes are provided for use with APIs in the Azure SDK for JavaScript that accept an `AbortSignalLike`
-parameter to cancel an operation.
+The `@azure/abort-controller` package provides `AbortController` and `AbortSignal` classes. These classes are compatible
+with the [AbortController](https://developer.mozilla.org/en-US/docs/Web/API/AbortController) built into modern browsers
+and the `AbortSignal` used by [fetch](https://www.npmjs.com/package/fetch).
```

Comment:
the `fetch` link should point to https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API